netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -28,10 +36,13 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
 //      public void Save(string assemblyFileName, System.Reflection.PortableExecutableKinds portableExecutableKind, System.Reflection.ImageFileMachine imageFileMachine) { }
         public void SetCustomAttribute(System.Reflection.ConstructorInfo con, byte[] binaryAttribute) { }
         public void SetCustomAttribute(System.Reflection.Emit.CustomAttributeBuilder customBuilder) { }
+        public void SetEntryPoint(System.Reflection.MethodInfo entryMethod) { }
+        public void SetEntryPoint(System.Reflection.MethodInfo entryMethod, System.Reflection.Emit.PEFileKinds fileKind) { }
```

Comment:
I don't believe `fileKind` is exposed anywhere, so I'm inclined to exclude this as we're excluding all persistence APIs.

Anybody else?

Reply:
Agree.

Reply:
👍